sub-italian-page.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Sub Italian Categories</title>
+    <style>
```

Comment:
Since you created a style file, why not use it instead of defining styles in the style header here?